pkg/handler/unrouted_handler.go

```diff
@@ -1013,6 +1013,17 @@ func (handler *UnroutedHandler) GetFile(w http.ResponseWriter, r *http.Request)
 		return
 	}
 
+	// If the data store implements ContentServerDataStore, use the ServableUpload interface
+	if handler.composer.UsesContentServer {
+		servableUpload := handler.composer.ContentServer.AsServableUpload(upload)
+		err = servableUpload.ServeContent(c, w, r)
```

Comment:
I wonder if the handler should set the Content-Length, Content-Type and Content-Disposition before passing it to `ServeContent`. The implementation can then decide to overwrite or keep them. The idea is that we already have logic for extracting the filename and filetype from metadata and this way we can keep using them.

Reply:
you decide?

Reply:
Just implemented this by myself.

Reply:
Sorry, huh? where?